libr/core/cmd_seek.c

```diff
@@ -138,341 +138,401 @@ R_API int r_core_lines_initcache (RCore *core, ut64 start_addr, ut64 end_addr) {
 	return -1;
 }
 
+static void seek_to_register(RCore *core, const char *input, bool is_silent) {
+	ut64 off;
+	if (core->io->debug) {
+		off = r_debug_reg_get (core->dbg, input);
```

Comment:
you can directly define that ut64 off in the assignment line, and earn 1 line for free!!